cloudify_ansible/utils.py

```diff
@@ -134,6 +138,29 @@ def _get_tenant_name(_ctx=None):
     return _ctx.tenant_name
 
 
+def _get_collections_location(_ctx=None):
+    _ctx = _ctx or ctx
+    runtime_properties = get_instance(ctx).runtime_properties
+    if not is_local_venv() and \
+            (get_node(ctx).properties.get('galaxy_collections')
+             or runtime_properties.get('galaxy_collections')):
+        return runtime_properties.get(WORKSPACE)
+    return os.path.join(runtime_properties.get(PLAYBOOK_VENV),
+                        'lib',
+                        'python' + sys.version[:3],
+                        'site-packages'
+                        )
+
+
```

Comment:
We can just send the `ctx.instance`/`ctx.source/target.instance` object to `_get_roles_location` IMO, since we are not using other CTX objects, e.g. logger, node, etc.